.github/workflows/release.yml

```diff
@@ -172,16 +172,22 @@ jobs:
         ADO_AZUREAUTH_LINUX_PIPELINE_ID: ${{ secrets.ADO_AZUREAUTH_LINUX_PIPELINE_ID }}
         ADO_AZUREAUTH_LINUX_STAGE_ID: ${{ vars.ADO_AZUREAUTH_LINUX_STAGE_ID }}
         VERSION: ${{ github.event.inputs.version }}
-    - name: Upload linux-x64 artifact
-      uses: actions/upload-artifact@v3
-      with:
-        name: azureauth-${{ github.event.inputs.version }}-linux-x64.deb
-        path: ${{ env.ADO_LINUX_ARTIFACT_DOWNLOAD_PATH }}/${{ env.ADO_LINUX_ARTIFACT_NAME }}/azureauth_${{ github.event.inputs.version }}-${{ env.DEBIAN_REVISION }}_amd64.deb
-    - name: Upload linux-arm64 artifact
+
+    - name: Rename linux-x64 artifact
+      run: |
+        mv ${{ env.ADO_LINUX_ARTIFACT_DOWNLOAD_PATH }}/${{ env.ADO_LINUX_ARTIFACT_NAME }}/azureauth_${{ github.event.inputs.version }}-${{ env.DEBIAN_REVISION }}_amd64.deb \
+        azureauth-${{ github.event.inputs.version }}-linux-x64.deb
+    - name: Rename linux-arm64 artifact
+      run: |
+        mv ${{ env.ADO_LINUX_ARTIFACT_DOWNLOAD_PATH }}/${{ env.ADO_LINUX_ARTIFACT_NAME }}/azureauth_${{ github.event.inputs.version }}-${{ env.DEBIAN_REVISION }}_arm64.deb \
+        azureauth-${{ github.event.inputs.version }}-linux-arm64.deb

```

Comment:
We do the other actions involving these two artifacts in a single step. Can we do the same here as well?